libraries/authoritative_helpers.rb

```diff
@@ -0,0 +1,62 @@
+#
+# Cookbook Name:: pdns
+# Libraries:: authoritive_helpers
+#
+# Copyright 2014-2017 Aetrion LLC. dba DNSimple
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+def default_authoritative_config_directory
+  case node['platform_family']
+  when 'debian'
+    '/etc/powerdns'
+  when 'rhel'
+    '/etc/pdns'
+  end
+end
+
+def default_authoritative_run_user
```

Comment:
Since the cases here are identical, I'd just return 'pdns' for now in this method. Why add logic where it adds no value unless you intend to return nil.